src/python/pants/help/help_formatter_test.py

```diff
@@ -60,7 +60,7 @@ def test_format_help_choices(self) -> None:
     @classmethod
     def _get_parser(cls) -> Tuple[Parser, NativeOptionParser]:
         return Parser(
-            scope_info=GlobalOptions.get_scope_info(),
+            scope=GlobalOptions.options_scope,
```

Comment:
We don't need "ScopeInfo" in some places now, and hopefully in all places soon...